README.md

```diff
@@ -115,31 +113,20 @@ This repository is also configured with "EditorConfig" to make sure everyone's f
 
 Below are some useful commands that can be executed from the root of this repository in your terminal.
 
-- `./gradlew clean`: Removes `build` directory so that you have a fresh build result.
+- `./gradlew clean`: Removes `build` and `dist` directory so that you have a fresh build result.
+
+- `./gradlew rollup`: Bundles the `.js` files for NPM release.
 
 - `./gradlew unit_test`: Runs the mocha unit tests for the source files in `src` directory. Generates reports in `build/test-report` and `build/coverage-report` directories as well as your terminal.
 
 -  `./gradlew lint`: Runs the eslint for all `.js` files. Generates reports in `build/lint-report` directory as well as your terminal.
 
 - `./gradlew test`: Runs all test related gradle tasks, including `unit_test` and `lint`.
 
-
-## Maintenance
-
-This plugin currently is not published it to an npm registry. This section of the document is for maintainers of this repo on process for maintaining changes and release process.
-
 ### Release
 
-[Semantic Versioning](https://semver.org/) must be followed. When a new release is required, create a Git Tag and a release in GitHub following semantic versioning schema.
-
-- In GitHub repo, click "Create a new release"
-- In the "New Release" page:
-    - From the "Choose a tag" dropdown, type a new release version; for example: `v1.0.0`.
-    - Type the same version number as "Release Title"
-    - Give a brief description in the text box. Ideally a list of PRs with features/fixes added and the corresponding contributor's name.
-    - Click "Publish Release".
+Publishing to NPM and releasing in GitHub is automated by the CI/CD pipeline in this repo, and uses [Semantic Release](https://github.com/semantic-release/semantic-release), commit messages must follow the format described in the documentation for a release to be triggered.
 
-The above steps will create a new git tag as well as create a new GitHub release.
-
-Ideally we'd have the above steps automated and is taken care of through GitHub Actions as part of CI CD process.
+## Maintenance
 
+This plugin is published to the [npm registry under `@elasticpath` organization](https://www.npmjs.com/package/@elasticpath/remarkable-oembed). This section of the document is for maintainers of this repo on process for maintaining changes and release process.
```

Comment:
Reading the contents of this section, there isn't really any info for "maintainers". So, maybe let's just remove it.

Reply:
Removed.